src/main/resources/static/api.yaml

```diff
@@ -88,4 +172,122 @@ components:
       properties:
         hipId:
           type: string
-          description: Identifier of the health information provider where patient record needs to be looked up.
\ No newline at end of file
+          description: Identifier of the health information provider where patient record needs to be looked up.
+
+    ConsentRequestsRepresentation:
+      type: object
+      properties:
+        limit:
+          type: integer
+        size:
+          type: integer
+        offset:
+          type: integer
+        requests:
+          type: array
+          maxItems: 100
+          items:
+            $ref: '#/components/schemas/ConsentRequestRepresentation'
+
+    ConsentRequestRepresentation:
+      type: object
+      properties:
+        purpose:
+          type: object
+          properties:
+            description:
+              type: string
+        hitypes:
+          type: array
+          items:
+            $ref: '#/components/schemas/HITypeRepresentation'
+        requester:
+          type: object
+          properties:
+            id:
+              type: string
+              format: uuid
+              example: 496413f0-7b63-4f80-9bb0-9382cc3108d3
+            name:
+              type: string
+              example: "Dr. Karuppan"
+            orgName:
+              type: string
+              example: "TMH"
+            contact:
+              type: object
+              properties:
+                mobile:
+                  type: string
+                  example: '+9199999x9999'
+        duration:
+          type: object
+          properties:
+            from:
+              type: string
+              format: date-time
+            to:
+              type: string
+              format: date-time
+        expiry:
+          type: string
+          format: date-time
+        status:
+          type: string
+          enum:
+            - REQUESTED
+            - EXPIRED
+            - DENIED
+            - GRANTED
+
+    HITypeRepresentation:
+      type: object
+      properties:
+        type:
+          $ref: '#/components/schemas/HITypeEnum'
+        description:
+          type: string
+          example: HIType Description2
+
+    HITypeEnum:
+      type: string
+      enum:
+        - HIType1
+        - HIType2
+        - HIType3
+        - HIType4
+        - HIType5
+
+    ConsentApproveRequest:
+      type: object
+      properties:
+        consent:
```

Comment:
do we need this node "consent" within? 
it can be just 
{
    "hitypes": [
      "HIType1"
    ],
   permissions: { ... }
   .... 
}

instead of:  
"consent": {
    "hitypes": [
      "HIType1"
    ],
   ...
}

Reply:
Yes, that will keep the contract extendable.